metadata.rb

```diff
@@ -50,6 +50,10 @@
           :description => "Download url for Amazon ECR Credential Helper",
           :type => 'string'
 
+attribute "cloud/init/ndb/data_memory_multiplier",
```

Comment:
Is the default 0.05 or 0.07?

Reply:
I changed it to 5% because 7% was already too much - it's 2.2G

I forgot to change the default here, I'll change it